src/Features/Core/Portable/QuickInfo/Presentation/QuickInfoOnTheFlyDocsElement.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.QuickInfo.Presentation;
+
+internal sealed class QuickInfoOnTheFlyDocsElement(Document document, OnTheFlyDocsInfo info) : QuickInfoElement
```

Comment:
📝 This was moved and adjusted from https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core/IntelliSense/QuickInfo/EditorFeaturesOnTheFlyDocsElement.cs.